src/fsharp/block.fs

```diff
@@ -0,0 +1,222 @@
+module Internal.Utilities.Library.Block
+
+open System.Collections.Immutable
+
+type block<'T> = ImmutableArray<'T>
```

Comment:
To follow .NET naming shouldn't this be Block<'T> and BlockBuilder<'t> with only the caml style aliases being lowercase?

Reply:
It doesn't matter to me really, but I was trying to follow how we define `array<'T>`. -> `type array<'T> = 'T[]`